adapter/service/scenectrl/routing_test.go

```diff
@@ -22,6 +22,11 @@ const (
 	sceneUnsupported = "movietime"
 )
 
+var sceneReportColorloop = scenectrl.SceneReport{
```

Comment:
It's used only in TestRouteSceneCtrl so you can probably move this var declaration there.

Reply:
Done